src/after/DroneDelivery-after/Services/PackageServiceCaller.cs

```diff
@@ -18,7 +19,11 @@ public PackageServiceCaller(HttpClient httpClient)
 
         public async Task<PackageGen> CreatePackageAsync(PackageInfo packageInfo)
         {
-            var result = await httpClient.PutAsJsonAsync($"{packageInfo.PackageId}?code={FunctionCode}", packageInfo);
+            string jsonString = JsonConvert.SerializeObject(packageInfo);
+
+            StringContent httpContent = new StringContent(jsonString, System.Text.Encoding.UTF8, "application/json");
```

Comment:
Prefer `var` in instantiation cases like this where the type is obvious.

```suggestion
            var httpContent = new StringContent(jsonString, System.Text.Encoding.UTF8, "application/json");
```